src/panels/config/zha/zha-device-page.ts

```diff
@@ -15,75 +14,82 @@ import {
-import { ZHAClusterSelectedParams, ZHADeviceSelectedParams } from "./types";
+import { ZHAClusterSelectedParams } from "./types";
 
-export class HaConfigZha extends LitElement {
+@customElement("zha-device-page")
+export class ZHADevicePage extends LitElement {
   @property() public hass?: HomeAssistant;
   @property() public isWide?: boolean;
-  @property() private _selectedDevice?: ZHADevice;
+  @property() public ieee?: string;
+  @property() public device?: ZHADevice;
   @property() private _selectedCluster?: Cluster;
   @property() private _bindableDevices: ZHADevice[] = [];
 
   protected updated(changedProperties: PropertyValues): void {
-    if (changedProperties.has("_selectedDevice")) {
-      this._fetchBindableDevices();
+    if (changedProperties.has("ieee")) {
+      this._fetchData();
     }
     super.update(changedProperties);
   }
 
   protected render(): TemplateResult | void {
     return html`
-      <hass-subpage header="Zigbee Home Automation">
-        <zha-network
+      <hass-subpage
+        header="${this.hass!.localize("ui.panel.config.zha.devices.header")}"
+      >
+        <zha-node
           .isWide="${this.isWide}"
           .hass="${this.hass}"
-        ></zha-network>
-
-        <zha-groups-tile
-          .isWide="${this.isWide}"
+          .device=${this.device}
+        ></zha-node>
+        <zha-clusters
           .hass="${this.hass}"
-        ></zha-groups-tile>
-
-        <zha-node
           .isWide="${this.isWide}"
-          .hass="${this.hass}"
+          .selectedDevice="${this.device}"
           @zha-cluster-selected="${this._onClusterSelected}"
-          @zha-node-selected="${this._onDeviceSelected}"
-        ></zha-node>
+        ></zha-clusters>
         ${this._selectedCluster
           ? html`
               <zha-cluster-attributes
                 .isWide="${this.isWide}"
                 .hass="${this.hass}"
-                .selectedNode="${this._selectedDevice}"
+                .selectedNode="${this.device}"
                 .selectedCluster="${this._selectedCluster}"
               ></zha-cluster-attributes>
 
               <zha-cluster-commands
                 .isWide="${this.isWide}"
                 .hass="${this.hass}"
-                .selectedNode="${this._selectedDevice}"
+                .selectedNode="${this.device}"
                 .selectedCluster="${this._selectedCluster}"
               ></zha-cluster-commands>
             `
           : ""}
-        ${this._selectedDevice && this._bindableDevices.length > 0
+        ${this._bindableDevices.length > 0
           ? html`
               <zha-binding-control
                 .isWide="${this.isWide}"
                 .hass="${this.hass}"
-                .selectedDevice="${this._selectedDevice}"
+                .selectedDevice="${this.device}"
                 .bindableDevices="${this._bindableDevices}"
               ></zha-binding-control>
             `
           : ""}
+        <div class="spacer" />
```

Comment:
Can't you give a margin to the last element in `hass-subpage`?
```css
hass-subpage > *:last-child {
    margin-bottom: 50px;
}
```

Reply:
sure